routes/auth/sign-in.js

```diff
@@ -0,0 +1,151 @@
+const express = require("express");
+const axios = require("axios");
+
+const admin = require("../../config/firebase-config");
+const { User } = require("../../model/User");
+
+const router = express.Router();
+
+const {
+  generateAccessToken,
+  generateRefreshToken,
+} = require("../../utils/generateToken");
```

Comment:
```suggestion
const {
  generateAccessToken,
  generateRefreshToken,
} = require("../../utils/generateToken");

const router = express.Router();
```

소소하지만 호출 순서가 이렇게 되어야 할 것 같습니다.
(require 문끼리 먼저 호출이 된 이후에 router() 호출)

Reply:
변경처리 하였습니다!👌